interface/menu.dm

```diff
@@ -76,6 +76,23 @@ GLOBAL_LIST_EMPTY(menulist)
 /datum/verbs/menu/Icon/Size
 	checkbox = CHECKBOX_GROUP
 	default = /datum/verbs/menu/Icon/Size/verb/iconstretchtofit
+/datum/verbs/menu/Icon/Size/Load_checked(client/C)
+	var/atom/verb/verbpath = Get_checked(C)
```

Comment:
byond acts odd if you don't. it doesn't matter for this use case, but if you want to access `verbpath.name` from a verb path, it has to be casted as a verb.

verb path and verb object are one in the same.

(or would you rather i do `verbpath:name` =P)